_src/pldoc/static/sass/_build.scss

```diff
@@ -24,6 +24,7 @@
 @import '../../../pattern-library/sass/components/buttons';
 @import '../../../pattern-library/sass/components/headings';
 @import '../../../pattern-library/sass/components/copy';
+@import '../../../pattern-library/sass/components/forms';
```

Comment:
@clrux, can you update the Sass/CSS styleguide compilation examples with this new partial?